src/server/boot/03-participant.js

```diff
@@ -105,7 +117,7 @@ export default function (app) {
         req.body.ids,
         req.body.fieldName,
         req.body.newValue,
-        req.user.id,
+        req,
```

Comment:
this should also be a call to `getClientData`

Reply:
Done